core/server/api/canary/members.js

```diff
@@ -432,16 +484,12 @@ module.exports = {
             const memberLabels = serializeMemberLabels(getUniqueMemberLabels(frame.data.members));
             await findOrCreateLabels(memberLabels, frame.options);
 
-            return Promise.resolve().then(() => {
-                const {sanitized, duplicateStripeCount} = sanitizeInput(frame.data.members);
-                invalid.count += duplicateStripeCount;
+            return Promise.resolve().then(async () => {
+                const {sanitized, invalidCount, validationErrors} = await sanitizeInput(frame.data.members);
```

Comment:
This refactor in returned values allows to keep the invalid record count self contained and not have to deal with error types.